### DISCUSSION TITLE
VSCode Debugging Timeout (1.53.0)

### DISCUSSION THREAD

**Discussion by vicatcu** [avajs/ava#2678] [Q&A] [answered]
Is anyone experiencing this problem debugging in vscode where about 10s after starting a session, the debugger terminates with a message about an idle timeout? Before raising it as an issue, I just wanted to find out if it's already known or if there are known workarounds. Maybe just some kind of update is needed for this page https://github.com/avajs/ava/blob/main/docs/recipes/debugging-with-vscode.md. I don't think this used to be a problem for me, maybe something has changed in VSCode recently?

> Version: 1.53.0
> Commit: 8490d3dde47c57ba65ec40dd192d014fd2113496
> Date: 2021-02-03T15:56:04.185Z
> Electron: 11.2.1
> Chrome: 87.0.4280.141
> Node.js: 12.18.3
> V8: 8.7.220.31-electron.0
> OS: Linux x64 5.8.0-7630-generic snap

**Comment by novemberborn** [comment, marked as answer]
Which AVA version are you using?

**Reply by vicatcu** [reply]
Looks like 3.8.2 according to `npx ava --version`

**Reply by novemberborn** [reply]
You're going to have to use the latest.

**Reply by vicatcu** [reply]
Thanks @novemberborn, you were right, updating to the latest AVA version seems to have worked
